### DISCUSSION TITLE
migration error for sqlite

### DISCUSSION THREAD

**Discussion by zhangzhishan** [SeaQL/sea-orm#2392] [Q&A]
I am use a local sqlite file as the db, for example abc.db, when I run the migration it shows below error

```
thread 'main' panicked at C:\Users\root\scoop\persist\rustup\.cargo\registry\src\index.crates.io-6f17d22bba15001f\sea-orm-migration-1.0.1\src\cli.rs:30:10:
Fail to acquire database connection: Conn(Internal("The connection string 'sqlite://abc.db' has no supporting driver."))
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\migration.exe up -u sqlite://abc.db` (exit code: 101)
```

I have below settings in .env

```
DATABASE_URL=sqlite://abc.db
```

**Comment by DevWonder01** [comment]
![image](https://github.com/user-attachments/assets/7c9d7b41-49db-4719-951a-bae6303efb76)
you will need to uncomment this 
  # "runtime-tokio-rustls",  # `ASYNC_RUNTIME` feature
  # "sqlx-postgres",         # `DATABASE_DRIVER` feature
  
  to 
  
   "runtime-tokio-rustls",  # `ASYNC_RUNTIME` feature
   "sqlx-postgres",         # `DATABASE_DRIVER` feature
   
![image](https://github.com/user-attachments/assets/27b1e874-ff80-496b-a53e-70a1aec83a52)


this would fix it
